### DISCUSSION TITLE
关于 ImGuiAnt.FontAtlasBuild() 性能的问题

### DISCUSSION THREAD

**Discussion by constly** [ejoy/ant#112] [General]
我发现这个函数实在是太耗了，
在我电脑上：AMD Ryzen 9 5900X 12-Core Processor               3.70 GHz，需要执行1.12秒。
之前启动时卡一下也就忍了，现在加了切换场景的功能后，本来是秒切的，只要一 import_feature "ant.imgui"，每次切换场景就巨难受。

有没有办法优化下呢，比如场景切换前和切换后都有imgui，就不要先卸载，再重新初始化 之类的~

**Comment by cloudwu** [comment]
等下周一我们讨论一下。

**Comment by cloudwu** [comment]
这周在重构 ltask 的调度器。等弄完了再搞这个。

**Reply by constly** [reply]
OK，不着急~

**Comment by constly** [comment]
我看这块已经优化完成，问题可以关了~
